types.d.ts

```diff
@@ -435,7 +440,7 @@ export declare class Molecule {
    */
   getOCL(): any;
 
-  toSmiles(): string;
+  toSmiles(mode?: number): string;
```

Comment:
We never documented the possibility to have a boolean so I didn't make it a breaking change

Reply:
Yes they can be combined so this approach is better indeed.